addStyles.js

```diff
@@ -21,9 +21,7 @@ var stylesInDom = {},
 	styleElementsInsertedAtTop = [];
 
 module.exports = function(list, options) {
-	if(typeof DEBUG !== "undefined" && DEBUG) {
-		if(typeof document !== "object") throw new Error("The style-loader cannot be used in a non-browser environment");
```

Comment:
Scratch my comment. I can see the point now. Basically we have to wait for a decision before we can move on the PR.